rdr/busybox/cephfs/app-busybox-2/resources/deployment/busybox-dc-1.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    appname: busybox_app2_cephfs-1
+    workloadpattern: simple_io
+  name: busybox-1
+spec:
+  selector:
+    matchLabels:
+      workload: busybox-1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        workload: busybox-1
+    spec:
+      containers:
+      - command: ["/bin/sh", "-c"]
+        args:
+          - wget  https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/master/rdr/busybox/scripts/simple_io.sh -O /mnt/test/simple_io.sh;
```

Comment:
how about updating our image `rdr-ocs-workload` to come prepared with simple_io.sh and simple_io_integrity_check.sh, instead of downloading them each time?
Should save execution time

Reply:
If we change the script, we need to rebuild the image again and again, which is why we don't do that .

Reply:
Also, 1 more concern is that we support multi-arch images and cannot quickly build directly from the quay. We have to use the local laptop for that

Reply:
Ok, understood